src/test/java/trade/wayruha/whitebit/service/AccountServiceTest.java

```diff
@@ -0,0 +1,70 @@
+package trade.wayruha.whitebit.service;
+
+import org.junit.Test;
+import trade.wayruha.whitebit.TestConstants;
+import trade.wayruha.whitebit.WBConfig;
+import trade.wayruha.whitebit.dto.DepositAddressResponse;
+import trade.wayruha.whitebit.dto.PageableResponse;
+import trade.wayruha.whitebit.dto.TransactionFees;
+import trade.wayruha.whitebit.dto.TransactionRecord;
+import trade.wayruha.whitebit.dto.request.TransactionHistoryRequest;
+import trade.wayruha.whitebit.dto.request.TransferRequest;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static trade.wayruha.whitebit.domain.enums.Account.MAIN;
+import static trade.wayruha.whitebit.domain.enums.Account.TRADE;
+
+public class AccountServiceTest {
+  WBConfig config = TestConstants.getSimpleConfig();
+  AccountService service = new AccountService(config);
+  final String usdt = "USDT";
+
+  @Test
+  public void test_getDepositAddress() {
+    final DepositAddressResponse address = service.getDepositAddress(usdt, "TRC20");
+    System.out.println("Address for " + usdt + ": " + address);
+  }
+
+  @Test
+  public void test_transfers() {
+    final BigDecimal qty = new BigDecimal("1");
+    final TransferRequest mainToTrade = new TransferRequest(MAIN, TRADE, usdt, qty);
+    final TransferRequest tradeToMain = new TransferRequest(TRADE, MAIN, usdt, qty);
+    service.transfer(mainToTrade);
+    System.out.println("Transferred " + qty + " " + usdt + " from main to trade");
```

Comment:
Is it sysout need for test or just left debug logs?

Reply:
it's not needed, just a kind of debug logs for the user to see the received object